ns/signaling_domain_zone.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+SIGNALING_DOMAIN="_signal.$DESEC_NS_NAME"
+
+# Create Signaling Domain zone and sign with provided key
+pdnsutil create-zone "$SIGNALING_DOMAIN" || exit  # quit if zone exists
+pdnsutil replace-rrset "$SIGNALING_DOMAIN" "@" SOA \
+    "${DESEC_NS_SIGNALING_DOMAIN_SOA_MNAME}. ${DESEC_NS_SIGNALING_DOMAIN_SOA_RNAME}. 0 86400 3600 2419200 3600"
+KEY_FILE=$(mktemp)
+echo "$DESEC_NS_SIGNALING_DOMAIN_ZONE_KEY_B64" | base64 -d > "$KEY_FILE"
+pdnsutil import-zone-key "$SIGNALING_DOMAIN" "$KEY_FILE"
+rm "$KEY_FILE"
```

Comment:
Best practice:
```sh
pdnsutil import-zone-key "$SIGNALING_DOMAIN" <(base64 -d <<< "$DESEC_NS_SIGNALING_DOMAIN_ZONE_KEY_B64")
```

Avoids secret showing up in process list.

Reply:
nice